generator/types.go

```diff
@@ -635,7 +640,8 @@ func (f *Field) Value() string {
 	case Slice:
 		return fmt.Sprintf("types.Slice(%s), nil", name)
 	case Array:
-		return fmt.Sprintf("types.Array(%s), nil", f.fieldVarAddress())
+		arrayLength := f.mustFindArrayLenght()
+		return fmt.Sprintf("types.Array(%s, %d), nil", f.fieldVarAddress(), arrayLength)
```

Comment:
just call `arrayLen(f)`. the intermediate variable is not really needed for anything else